.github/scripts/upload_tests_results.py

```diff
@@ -0,0 +1,342 @@
+#!/usr/bin/env python3
+
+import argparse
+import configparser
+import fnmatch
+import os
+import subprocess
+import sys
+import time
+import ydb
+import xml.etree.ElementTree as ET
```

Comment:
Probably specialized library is an better option for junit parsing? (first link in google: https://pypi.org/project/junitparser/ )

Reply:
We decided do that in further PR

Reply:
https://github.com/ydb-platform/ydb/issues/6430